docs/book/features/pv_to_backingdiskobjectid_mapping.md

```diff
@@ -0,0 +1,93 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+# PV to BackingDiskObjectId Mapping
+
+- [Introduction](#introduction)
+- [Prerequisite](#prereq)
+- [How to enable PV to BackingDiskObjectId Mapping feature in vSphere CSI](#how-to-enable)
+- [How to use PV to BackingDiskObjectId Mapping feature](#how-to-use)
+
+## Introduction <a id="introduction"></a>
+
+PV to BackingDiskObjectId Mapping feature will be added in vSphere CSI driver 2.5 as an Alpha feature. This feature provide backup partners with PV to BackingDiskObjectId mapping
+so they can do volume placement on the target side.
```

Comment:
Change "This feature provide backup partners with PV to BackingDiskObjectId mapping so they can do volume placement on the target side." to "This feature adds PV to BackingDiskObjectId mapping as an annotation on PVC".